README.md

```diff
@@ -467,57 +299,86 @@ pg_dump:
 PGHOST
 PGPORT
 PGUSER
+PGDATABASE
 ```
 
 It is also possible to configure access via pg_dump's command-line options
 `--host`, `--port`, and `--username`.
 
-### 2. Verify pg_dump output
+#### 1.2 Direct access to PostgreSQL
+
+In this case, HarbourBridge connects directly to the PostgreSQL
+database to retrieve table schema and data using the setting of
+standard _PGHOST_, _PGPORT_, _PGUSER_, _PGDATABASE_ environment
+variables. Password can be specified either in the _PGPASSWORD_
+environment variable or provided at the password prompt.
+
+#### 1.3 mysqldump
+
+If you are using mysqldump (-driver=mysqldump), check that mysqldump is
+correctly configured to connect to your MySQL
+database. Note that mysqldump uses the same options as mysql to connect to your
+database. See the [mysql](https://dev.mysql.com/doc/refman/8.0/en/mysql-commands.html) and
+[mysqldump](https://dev.mysql.com/doc/refman/8.0/en/mysqldump.html) documentation.
+
+Access to a MySQL database is configured via mysqldump's command-line options `--host`,
+`--port`, and `--user`.
+
+#### 1.4 Direct access to MySQL
+
+In this case, HarbourBridge connects directly to the MySQL
+database to retrieve table schema and data using the setting of
+_MYSQLHOST_, _MYSQLPORT_, _MYSQLUSER_, _MYSQLDATABASE_ environment
+variables. Password can be specified either in the _MYSQL_PWD_
```

Comment:
Why is there an underscore in MYSQL_PWD?
None of the others have it -- shouldn't this just be MYSQLPWD or MYSQLPASSWORD for consistency?

Reply:
`MYSQL_PWD` is among the few standard variable environment available for connecting to mysql. Rest of the variables as we had previously discussed are not standard because in MySQL connection is usually done by command line arguments.
Thus, I thought it would be a good idea to follow the standard and keep it as `MYSQL_PWD` instead of following the consistency and keeping it as `MYSQLPASSWORD`.

Reply:
This mixed style of environment variables looks confusing and inconsistent. We probably should agree on what kind of convention we should follow: 

1. Use the same style either `MYSQLPWD` or `MYSQL_PWD` for all environment variables. 
2. Environment variables should be consistent with the convention of the corresponding database, e.g., 
    * PostgreSQL: use _PGHOST_, _PGPORT_, _PGUSER_, _PGDATABASE_, _PGPASSWORD_. 
    * MySQL: use _MYSQL_HOST_, _MYSQL_PORT_, _MYSQL_USER_, _MYSQL_DATABASE_, _MYSQL_PWD_. 

(1) or (2)? 

I personally prefer (1).

Reply:
Ok sure. I also think (1) would be preferable. I will change it to `MYSQLPWD`.